src/components/text-input/index.md.njk

```diff
@@ -67,10 +67,20 @@ Fluid width inputs will resize with the viewport.
 
 ### Hint text
 
-Use hint for help that’s relevant to the majority of users, like how their information will be used, or where to find it.
+Use hint text for help that’s relevant to the majority of users, like how their information will be used, or where to find it.
 
 {{ example({group: "components", item: "text-input", example: "input-hint-text", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+#### When not to use hint text
+
+Do not use hint text for long paragraphs and lists. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+If your question contains paragraphs and lists, consider using a [question pages pattern](https://design-system.service.gov.uk/patterns/question-pages/) instead.
```

Comment:
I don't think the question pages pattern currently shows people how to do this.

I think the approach might be something like
- h1 with question
- paragraph / list to explain question
- input with visually-hidden question in label
- button

But we haven't shown that anywhere in an example as far as I'm aware.

So we could either try explain the general approach in guidance here (which might get complicated to explain) or add an example to the question page pattern (which might need some thought to make sure it makes sense). Or we leave it for now and be content the guidance is an improvement without needing to explain what to do instead!

Reply:
Thanks, @christopherthomasdesign! Have noted the approach.

Sounds like there are 2 issues here:
- advising that users avoid paragraphs/lists in hint text
- explaining how users can structure complex questions without using hint text

Maybe the best way forward is to update the hint text guidance for now, and meanwhile I'll create an issue to start off the pattern update? If so, should we remove the sentence here that directs user to the question pages pattern?

Reply:
Yeah I think I'd remove that sentence for now – because the link doesn't currently show you what to do instead.

Reply:
Thanks - have removed it!